addons/account/models/chart_template.py

```diff
@@ -1008,6 +1025,12 @@ def _load_translations(self, langs=None, companies=None):
         translation_importer = TranslationImporter(self.env.cr, verbose=False)
         for chart_template, chart_companies in groupby(companies, lambda c: c.chart_template):
             template_data = self.env['account.chart.template']._get_chart_template_data(chart_template)
+            for company in companies:
+                foreign_countries = self.env['account.tax'].with_context(active_test=False).search(
+                    [('company_id', '=', company.id)],
+                ).country_id - companies.country_id
+                if foreign_countries:
+                    template_data = self._add_foreign_taxes_data(company, foreign_countries, template_data)
```

Comment:
- business issue:
`companies.country_id`: I think it's possible that those companies might not have the same country despite having the same localization package.
Shouldn't we check for the `chart_template` country instead?

- performance issue: we should get the search out of the loop.
    - either we perform a first search and then a `filtered` (which might still be bad performance wise as it's n² (even n³ with the root loop))
    - either we perform a `.search(...).grouped('company_id')` but we end up with something to post-process as it will be a dict of tax per company and we want countries
    - either we do a `.read_group(...)` that might give us what we're looking for